client/web/src/Structure/Results Display/State/action.js

```diff
@@ -2,7 +2,6 @@ import { resultConstant } from "./actionTypes";
 import axios from "axios";
 
 const RESULT_API = process.env.REACT_APP_ATTEMPT_URL;
-const token = localStorage.getItem("token");
```

Comment:
use enums for token names... i have already created enum folder. Use that.
Also the localstorage methods are in utils, use that